src/handlers/store-standardised-invoices/handler.ts

```diff
@@ -24,23 +26,52 @@ export const handler = async (event: SQSEvent): Promise<Response> => {
   if (rawInvoiceBucket === undefined || rawInvoiceBucket.length === 0)
     throw new Error("Raw invoice bucket not set.");
 
+  for (const records of event.Records) {
+    if (!records || !records.body) {
+      throw new Error("Record contains no body");
+    }
+  }
+
+  let recordBodies: StandardisedLineItem[] = [];
+  for (const records of event.Records) {
```

Comment:
Can we merge the above for loop into this one?

Reply:
would rather keep the validation seperate to be honest, will update second one though as it's not needed as much as it's not got lots of logs in it now